components/shared/AppSidebar/AppSidebar.tsx

```diff
@@ -62,6 +59,24 @@ export const AppSideBar = () => {
     .slice(0, 5)
     .sort((a, b) => new Date(b.updated_at).getTime() - new Date(a.updated_at).getTime());
 
+  interface GlobalEventHandlersEventMap {
+    workspaceUpdated: CustomEvent;
+  }
+
+  useEffectOnce(() => {
```

Comment:
For my own knowledge, why `useEffectOnce` instead of `useEffect` with an empty array?

Reply:
Pretty much. It's an alias.